.github/actions/godot-install/action.yml

```diff
@@ -74,6 +74,7 @@ runs:
             rm -rf ${{ inputs.godot-cache-path }}/$GODOT_BIN*
           fi
 
+          ls -lsa ${{ inputs.godot-cache-path }}
           mv ${{ inputs.godot-cache-path }}/Godot_v${{ inputs.godot-version }}* ${{ inputs.godot-cache-path }}/godot
```

Comment:
What's the reason for using `Godot_v${{ inputs.godot-version }}*` instead of just using `GODOT_BIN`?
Correct me if I'm wrong, but I don't think it's named differently.

Reply:
there is a difference in the package for mono and default dist
so i need to have an another else here to fix